src/commands/sokoban.ts

```diff
@@ -0,0 +1,95 @@
+import { PathSrc } from '#lib/common/constants';
+import { LanguageKeys } from '#lib/i18n/LanguageKeys';
+import { buildGameControls, buildSokobanGameFromResolvableLevel, encodeResolvableLevel } from '#lib/utilities/sokoban';
+import { Command, RegisterCommand, RegisterSubcommand, type AutocompleteInteractionArguments } from '@skyra/http-framework';
+import { applyLocalizedBuilder, getSupportedLanguageT } from '@skyra/http-framework-i18n';
+import { MessageFlags } from 'discord-api-types/v10';
+import { readFile } from 'fs/promises';
+import { URL } from 'url';
+
+const Root = LanguageKeys.Commands.Sokoban;
+const Levels: Level[] = JSON.parse(await readFile(new URL('lib/common/levels.json', PathSrc), { encoding: 'utf-8' }));
+
+@RegisterCommand((builder) => applyLocalizedBuilder(builder, Root.RootName, Root.RootDescription))
+export class UserCommand extends Command {
+	@RegisterSubcommand((builder) =>
+		applyLocalizedBuilder(builder, Root.OptionPlayLevel) //
+			.addStringOption((builder) => applyLocalizedBuilder(builder, Root.OptionChooseLevel).setAutocomplete(true).setRequired(true))
+	)
+	public async level(interaction: Command.ChatInputInteraction, options: LoadLevelOptions) {
```

Comment:
The method should match the subcommand's name:
```suggestion
	public async playLevel(interaction: Command.ChatInputInteraction, options: LoadLevelOptions) {
```